.github/workflows/ci.yaml

```diff
@@ -67,50 +67,34 @@ jobs:
         cd functions/go
         make check-licenses
 
-  verify-ci:
-    runs-on: ubuntu-latest
-    env:
-      GOPATH: /home/runner/work/kpt-functions-catalog/functions/go
-      GO111MODULE: on
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up python 3
-      uses: actions/setup-python@v2
-    - name: Install libs
-      run: |
-        pip install pyyaml
-    - name: Set up Go 1.15
-      uses: actions/setup-go@v2
-      with:
-        go-version: 1.15
-    - name: Install mdrip
-      run: |
-        go get github.com/russross/blackfriday/v2@v2.0.1
-        go get github.com/monopole/mdrip@v1.0.2
-    - name: Verify docs
-      run: |
-        make verify-docs
-
   e2e-ci:
-    needs: [ts-unit-test-ci, go-unit-test-ci, verify-ci]
+    needs: [ts-unit-test-ci, go-unit-test-ci]
     strategy:
       matrix:
         platform: [ubuntu-latest]
         node-version: [12.x]
     runs-on: ${{ matrix.platform }}
+    env:
+      GOPATH: /home/runner/work/kpt-functions-catalog/functions/go
+      GO111MODULE: on
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
+    - name: Set up Go 1.15
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+    - name: Install kpt
+      run: |
+        go get github.com/GoogleContainerTools/kpt@19371100c672616db568813040de4c9adefeda3c # HEAD of next as of 05/12
```

Comment:
is there an issue for this?

Reply:
Given most behaviors of render and eval have become much more stable recently, I can change to use `next`:
`go get github.com/GoogleContainerTools/kpt@next`.